### DISCUSSION TITLE
Image component of next/image is giving 404 in production

### DISCUSSION THREAD

**Discussion by gargpuja** [vercel/next.js#21002] [Help]
Hi,
I am trying to use Image component where src of image is the S3 url , its working locally giving 404 in production.
```
<Image className="banner-image" width={1024} height={309} src="https://sy-media-store.s3-us-west-2.amazonaws.com/3edd3bf4d188fb8cb8734c7cde91ac4b/45d6918c7cbfc7ec24a1868ceef06c20/5b9659ef-0b5c-42c8-acb7-db6be7962627.jpg" alt="cover image" layout="responsive"/> 
```
I have mentioned domain in a file called  next.config.js
```
module.exports = {
images: {
    domains: ['sy-media-store.s3-us-west-2.amazonaws.com'],
  },
  }
 ```

**Comment by mark-a-bailey** [comment]
@gargpuja , can you tell us more about the deployment method and config please?

I am deploying via serverless next and I am also getting the same error. It looks like (for me anyway) that it is not updating CloudFormation templates for this route

**Comment by mark-a-bailey** [comment]
Also, as per @jamesmosier 's comment (https://github.com/vercel/next.js/discussions/20953#discussioncomment-273158), your images config shouldn't reference your subdomain, just the domain itself.
try

```
images: {
    domains: ['amazonaws.com'],
  },
```
